Sources/ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -140,11 +140,12 @@ extension SQLiteNormalizedCache: NormalizedCache {
     let result: Swift.Result<[Record?], Error>
     do {
       let records = try self.selectRecords(forKeys: keys)
+      let recordsIndexMap = records.indices.reversed().reduce(into: [:]) { recordsIndexMap, index in
+        recordsIndexMap[records[index].key, default: index] = index
```

Comment:
Sorry for the drive by code review...

I'd use `resultMap` or just `result` for the first closure parameter name, because it's a little confusing to read the parameter as the same name as the resulting variable

Reply:
👍 Will take care of it shortly.